lms/djangoapps/courseware/views/views.py

```diff
@@ -671,6 +671,21 @@ def course_about(request, course_id):
         return render_to_response('courseware/course_about.html', context)
 
 
+@ensure_csrf_cookie
+@cache_if_anonymous()
+def program_detail(request, program_id):
+    """
+    Display the program's detail page.
+
+    Assumes the program_id is in a valid format.
+    """
+    program = get_program_details(request.user, program_id)
+
+    if not program:
+        raise Http404
+    return render_to_response('courseware/program_detail.html', {'program': program})
```

Comment:
We should remove this view code and associated url specification from this PR add it to the combined PR, since we have not program_detail.html template here.